connectors/src/connectors/zendesk/temporal/worker.ts

```diff
@@ -63,5 +61,6 @@ export async function runZendeskWorkers() {
     },
   });
 
-  await gcWorker.run();
+  // the run is blocking, we need to launch both workers in parallel
+  await Promise.all([syncWorker.run(), gcWorker.run()]);
```

Comment:
Why do we want 2 workers on the same pod vs just one worker ?

Reply:
compared to Notion the workload induced by the GC worker should be relatively low so it seemed more simple to put both on the same pod vs adding a deployment

Reply:
@flvndvd I'll let you follow up on that

Reply:
And the 2 different workers is to have different concurrency for GC and sync activities ?

Reply:
yeah the idea was that GC workflow runs once a day but will most likely be blocking for quite a while, this way we still have syncs running in the meantime

Reply:
not sure how important this is in practice though, open to any suggestion here

Reply:
Actually: https://github.com/dust-tt/dust/pull/8830